src/signal-handling.c

```diff
@@ -53,13 +53,6 @@ JL_DLLEXPORT void jl_exit_on_sigint(int on)
     exit_on_sigint = on;
 }
 
-// what to do on SIGINT
-JL_DLLEXPORT void jl_sigint_action(void)
```

Comment:
I don't see any registered Julia packages ccall'ing into it. Who knows about unregistered or embedding code though.